src/triggerReader_cross.cc

```diff
@@ -341,7 +341,7 @@ bool triggerReader_cross::checkOREleEleNew  (Long64_t triggerbit_1, Long64_t mat
       {
 	firedPath = _allTriggers.at(_eeTriggers.at(i));
 	boost::regex re_tau1{"Ele(\\d+)"};
-	ptCut = checkPtCutSingle(thisPath, firedPath, re_tau1, pt_tau1, 1.0); // 1.0 GeV from HTT twiki Run2 Legacy analisys
+	ptCut = checkPtCutSingle(thisPath, firedPath, re_tau1, pt_tau1, 2.0);
```

Comment:
Where does this 2 GeV threshold come from?

Reply:
Ok thanks! If there is no mention in the tau trigger twiki than it's better to leave it as it is.